hack/release.toml

```diff
@@ -25,6 +25,13 @@ preface = """
 * runc: 1.2.1
 
 Talos is built with Go 1.23.3.
+"""
+
+    [notes.cgroupsv1]
+        title = "cgroups version 1 deprecation"
+        description = """\
+Add a warning when `talos.unified_cgroup_hierarchy=0` parameter is passed to notify users of the deprecation.
```

Comment:
Support for `cgroupsv1` is deprecated, and will be removed in Talos 1.10 (for non-container mode).